joint1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_1
    joint_speed: 2.0
    motor:
        id: 1
        init: 1023
        min: 0
        max: 4095

joint2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_2
    joint_speed: 2.0
    motor:
        id: 2
        init: 1023
        min: 0
        max: 4095

joint3_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_3
    joint_speed: 2.0
    motor:
        id: 3
        init: 1023
        min: 0
        max: 4095


